projects/storefrontstyles/scss/layout/_index.scss

```diff
@@ -15,6 +15,7 @@
 @import './page-templates/search-results-grid';
 @import './page-templates/order-confirmation-page';
 @import './page-templates/myaccount-v2-page';
+@import './page-templates/main';
```

Comment:
I think we don't have a `main` template. Maybe we can include those styles in `_storefront.scss`?

Reply:
@Pio-Bar fixed. thanks for suggestion 👍